client/src/stylesheets/Post.css

```diff
@@ -37,29 +55,34 @@
     padding: 12px 20px;
     margin-bottom: 20px;
     margin-left: 10px;
+    font-size: medium;
 }
 
 .body-input {
-    width: 80%;
+    width: 80vw;
+    max-width: 912px;
     word-break: break-word;
     height: 130px;
     padding: 12px 20px;
     box-sizing: border-box;
     margin-left: 10px;
     margin-bottom: 2px;
     resize: none;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    font-size: medium;
 }
 
 .submit-input {
-    float: right;
     border: none;
-	padding: 8px;
+    margin-left: 10px;
+    padding: 8px;
+    font-size: medium;
 	background: #FF8500;
 	color: #fff;
```

Comment:
I think you already mentioned that Rails generate a lot of this but the indentation is a little bit inconsistent. 2 spaces in some places and 4 in others. Not something that has to be done for this PR but prettier would help fix all this up for you: https://create-react-app.dev/docs/setting-up-your-editor/#formatting-code-automatically

Reply:
Its a pain 🎃 I installed Prettier extension but didnt know how to get it started, I will look into this now

Reply:
Yep the extension will be fine since you're the only one working on this repo, but if there's were multiple people you'd want to actually install it as a tool so everyone's setup is the same. In VS Code you can also enable "format on save" which is handy as it's runs prettier every time you save the file.

Reply:
Using it now, really nice